docker/kites/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:latest
```

Comment:
What's the size of this image after you build it?

Can you evaluate an [Alpine-based](https://hub.docker.com/_/alpine) solution? We don't have hard dependencies on glibc I suppose, so worth exploring.

Reply:
I try now with Alpine which is certainly smaller, the whole image now is around 70MB

Reply:
of course the smaller the image, the better... but I am afraid that netsniff-ng as a package will not work (I would be happy to be wrong :-) ) @simonezacca: give it a quick try, otherwise move on with debian, then we can add an issue tagged as enhancement to look for a smaller footprint solution

Reply:
I quickly checked an didn't find any major blockers for Alpine and netsniff.

Reply:
Right now netsniff-ng is not available in the Alpine repository, it is the only package that gives problems during the build of the image.

This would be the new image:
`FROM alpine:3.8

RUN apk update && apk add --no-cache\
    iperf3\
    netsniff-ng\
    nano\
    tcpdump\
    screen

EXPOSE 5201

ENTRYPOINT ["iperf3"]
CMD ["-s"]
`

But with netsniff-ng return:
`Sending build context to Docker daemon  2.048kB
Step 1/5 : FROM alpine:3.8
 ---> c8bccc0af957
Step 2/5 : RUN apk update && apk add --no-cache    iperf3    netsniff-ng    nano    tcpdump    screen
 ---> Running in e42f2529b608
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz
v3.8.5-16-ge61670a866 [http://dl-cdn.alpinelinux.org/alpine/v3.8/main]
v3.8.5-12-gdfb0d07a73 [http://dl-cdn.alpinelinux.org/alpine/v3.8/community]
OK: 9563 distinct packages available
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  netsniff-ng (missing):
    required by: world[netsniff-ng]`

without return built image